dashboard/app/models/course_version.rb

```diff
@@ -0,0 +1,24 @@
+# == Schema Information
+#
+# Table name: course_versions
+#
+#  id                 :integer          not null, primary key
+#  version_name       :string(255)
```

Comment:
Same questions about using key and display name?

Reply:
Do versions need the ability to have a display name? I was thinking they would mostly be things like "2020" and not be renamed / customized beyond that.

I could see having this for flexibility, but wondering if we already need this.

Reply:
So far I think they have been tied to year so renaming hasn't been needed but I wonder if at some time we will get to a point where users pick their version naming? Like 'Fall 2020' and 'Spring 2021'. I think its fine if we don't need it now to go without it but just wanted to ask

Reply:
yes, they do need display names. CSF doesn't use these, but CSP does:
![Screen Shot 2020-07-17 at 11 35 49 AM](https://user-images.githubusercontent.com/8001765/87819842-c999d780-c821-11ea-960d-8fd94bf57341.png)